azure-pipelines.yml

```diff
@@ -148,3 +148,33 @@ stages:
               ./test.sh --configuration $(_BuildConfig) --ci --test --integrationTest --nobl
             name: Test
             displayName: Tests
+
+      - job: MacOS
+        timeoutInMinutes: 90
+        pool:
+          name: Azure Pipelines
+          vmImage: macos-latest
+          os: macOS
+        strategy:
+          matrix:
+            Release:
+              _BuildConfig: Release
+            Debug:
+              _BuildConfig: Debug
+        steps:
+        - script: eng/common/cibuild.sh
+            -configuration $(_BuildConfig)
+            -prepareMachine
+            /p:Test=false
+            /p:NonWindowsBuild=true
```

Comment:
Is this property used anywhere?

Reply:
No idea, it's copied from arcade guidelines.